netty/src/main/java/io/grpc/netty/NettyServer.java

```diff
@@ -157,7 +157,7 @@ public void start(ServerListener serverListener) throws IOException {
     ServerBootstrap b = new ServerBootstrap();
     b.group(bossGroup, workerGroup);
     b.channel(channelType);
-    if (NioServerSocketChannel.class.isAssignableFrom(channelType)) {
+    if (ServerSocketChannel.class.isAssignableFrom(channelType)) {
```

Comment:
Actually, why do we need this `if` at all?

Reply:
not really. it could suppress some warning on netty side but since NCT is not checking, i'll remove it.